server/routes/users/auth.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const passport = require('passport');
+
+const router = express.Router();
+
+router.post('/login', passport.authenticate('local', {
+  successRedirect : '/profile',
+  failureRedirect : '/login'
+}), 
+(req, res) => {
+  res.json({ success : true });
+});
+
+router.post('/logout', passport.authenticate('local', {
```

Comment:
Logout routes don't need the authenticate middleware. That would be like you're trying to authenticate a user's username and password when they are logging out.